<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vova-Lenovo-ideapad-310-15IKB" tests="5" failures="5" timestamp="28 Aug 2019 07:20:05 GMT" time="1.251" errors="0">
  <testcase name="changeBearNameWithPutRequest" time="1.143" classname="ChangeBearNameWithPutRequest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
	at ChangeBearNameWithPutRequest.changeBearNameWithPutRequest(ChangeBearNameWithPutRequest.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- changeBearNameWithPutRequest -->
  <testcase name="noValuesInFields" time="0.025" classname="EmptyValuesInAllFields">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
	at EmptyValuesInAllFields.noValuesInFields(EmptyValuesInAllFields.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- noValuesInFields -->
  <testcase name="nonExistentBear" time="0.023" classname="PutRequestForNonExistentBear">
    <failure type="java.lang.AssertionError" message="expected [404] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [500]
	at PutRequestForNonExistentBear.nonExistentBear(PutRequestForNonExistentBear.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- nonExistentBear -->
  <testcase name="wrongFieldNameBearName" time="0.025" classname="WrongFieldNameBearName">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
	at WrongFieldNameBearName.wrongFieldNameBearName(WrongFieldNameBearName.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- wrongFieldNameBearName -->
  <testcase name="wrongNameOfFielBearAgedUsingPutRequest" time="0.017" classname="WrongNameOfFieldBearAgeUsingPutRequest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
	at WrongNameOfFieldBearAgeUsingPutRequest.wrongNameOfFielBearAgedUsingPutRequest(WrongNameOfFieldBearAgeUsingPutRequest.java:20)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- wrongNameOfFielBearAgedUsingPutRequest -->
</testsuite>
